### DISCUSSION TITLE
How do we properly submit changes to playercore.js?

### DISCUSSION THREAD

**Discussion by ghost** [textadventures/quest#1245] [Developing Quest]
When we make changes to **PlayerController\playercore.js**, should we also submit **WebPlayer\playercore.js** with the same changes in the PR, or is the former enough, since the latter is created when you build the project?

---
I always forget it. Luckily, GitHub Desktop is there to remind me.

![image](https://github.com/user-attachments/assets/5508e27c-55ef-42de-8fc4-264b667507f6)

**Comment by KVonGit** [comment]
It seems best to just include this, too. 👍🏼 👍🏼
